crates/modules/logger/src/lib.rs

```diff
@@ -49,51 +64,68 @@ impl TryFrom<&ModuleConfig> for Config {
         Ok(Self {
             console: config.with_default("console", true)?,
             // file: config.required("file")?,
-            // syslog: config.required("syslog")?,
+            syslog: config.with_default("syslog", true)?,
```

Comment:
you should update also the `README` to include this default

Reply:
done!